src/frontend/src/pages/HomePage/components/TeamTypeSection.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Grid, Typography } from '@mui/material';
+import { TeamType } from 'shared';
+import { useState, useEffect } from 'react';
+import { getImageUrl } from '../../../utils/image.utils';
+
+interface TeamTypeSectionProps {
+  teamType: TeamType;
+}
+
+const TeamTypeSection = ({ teamType }: TeamTypeSectionProps) => {
+  const [imageUrl, setImageUrl] = useState<string>('');
```

Comment:
Why does this have to be a usestate if its not going to change?

Reply:
I made it a useState because it gets reset every time the page renders like on line 14. Is there a better way to do this.

Reply:
Ok I see what you mean. Generally you really want to avoid using a setState within a useEffect - I think you can achieve this using useRef

Reply:
You could declare the imageUrl like this:

`const imageUrlRef = useRef<string>('');`
And then you can assign values to `imageUrlRef.current`

Reply:
Honestly its pretty tricky and you might wanna do some googling/chatgpt but i think that would work. Lmk if you have questions

Reply:
Okay yeah i just tried that and i realized the issue comes because each division is a tab and that does't trigger a rerender. so that means once a picture shows up for one of the divisions, then it stays the same image even if I switch tabs because thats not considered a refresh. Is there another way I can do this